### DISCUSSION TITLE
An error occurred while applying Student Developer Pack

### DISCUSSION THREAD

**Discussion by YaoSiQian** [community/community#134360] [GitHub Education]
### Hi everyone,

When I tried to renew my Student Developer Pack (second application), I was denied with the following conditions:
```
1. Please ensure that your academic affiliation document contains your first name exactly as it appears in your [GitHub billing information](https://github.com/settings/billing/payment_information). Variations of your name and nicknames are unacceptable.

2. Your image contains keywords indicating that it may be an acceptance letter. Only proof of current academic enrollment is acceptable.

3. You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name exactly as it appears in your academic affiliation document. Please do not use a variation of your name or a nickname. Once you have updated your profile information log out and log back into GitHub before re-applying.
```

First of all, I didn't understand the exact meaning of the second article, I applied using the admission letter my high school gave me on July 23, 2024 (containing **the school's official seal**, my legal name and the date in Chinese format). I applied by placing my cell phone next to the notification letter to show the English translation and the date in academic format (btw, for Chinese application materials, do I continue to have to provide my own translation), and all in all ended up failing the application.

I can provide more details if you need them.
![notice](https://github.com/user-attachments/assets/9808d268-e3d8-415c-9805-4641a89c6704)

With regard to articles 1 and 3, I feel a deep sense of injustice. As you can see, my legal name is in CJK script. In Chinese names, both first name and last name have only 1~2 Chinese characters. However, GitHub billing information will reject your application when the last name or first name has **_only one Chinese character_**.

![example](https://github.com/user-attachments/assets/3ff88d17-b45e-4ed5-964e-cede7b0f200b "An example about Zhang San")
The name is not short, the requirement is too long.

Also, as you've seen I've changed my GitHub username to my legal name, but the application is still not going through.

I find it hard to understand and it is my renewal application.

I hope you can come up with a solution ASAP, and I hope that this issue will be properly resolved as it involves more than just my interests.

**Comment by YaoSiQian** [comment]
I remembered that I used my English name when I first applied, and I hope I can be given a chance to change it.

**Comment by Shreyas719** [comment]
Check Student identity information then try it.

**Comment by YaoSiQian** [comment]
But the billing information problem is still there.And I've requested for one more time.
Waiting for the result.
&nbsp;


?????谦谦?????
***@***.***



&nbsp;




------------------&nbsp;Original&nbsp;------------------
From: "Shreyas ***@***.***&gt;; 
Date: 2024年8月2日(星期五) 中午11:11
To: ***@***.***&gt;; 
Cc: ***@***.***&gt;; ***@***.***&gt;; 
Subject: Re: [community/community] An error occurred while applying Student Developer Pack (Discussion #134360)














 
Check Student identity information then try it.
